pb/relay.proto

```diff
@@ -21,6 +21,7 @@ message CircuitRelay {
     STOP_DST_MULTIADDR_INVALID = 351;
     STOP_RELAY_REFUSED         = 390;
     MALFORMED_MESSAGE          = 400;
+    RELAY_OVERLOADED           = 500;
```

Comment:
Are we going to use this or should we just leave it at a reset?

Reply:
I will drop it in the rebase/squash.